js/components/dataset/card.vue

```diff
@@ -3,70 +3,52 @@
     <a v-if="dataset.organization" class="card-logo" :href="dataset.page">
         <img :alt="dataset.organization.name" :src="logo">
     </a>
+
     <img v-if="dataset.organization && dataset.organization.public_service"
         :src="certified" alt="certified" class="certified"
         v-popover="_('The identity of this public service is certified by %(certifier)s', certifier=config.SITE_AUTHOR)"
         :popover-title="_('Certified public service')"
         popover-trigger="hover"/>
+
     <div class="card-body">
         <h4>
             <a :href="dataset.page" :title="dataset.title">
-                {{ dataset.title | truncate 80 }}
+                {{ dataset.title }}
             </a>
         </h4>
+
+        <div class="clamp-3">{{{ dataset.description | markdown 180 }}}</div>
     </div>
-    <footer>
+
+    <footer class="card__footer">
         <ul>
-            <li v-if="dataset.spatial && dataset.spatial.zones && dataset.spatial.zones.length > 0">
-                <a class="btn btn-xs" v-tooltip tooltip-placement="top" :title="_('Territorial coverage')">
-                    <span class="fa fa-map-marker fa-fw"></span>
-                    {{ dataset.spatial.zones[0].name }}
-                </a>
+            <li v-tooltip :title="_('Resources count')">
+                <span class="fa fa-files-o fa-fw"></span>
+                {{ dataset.resources.length  }}
             </li>
-            <li v-if="dataset.metrics">
-                <a class="btn btn-xs" v-tooltip tooltip-placement="top" :title="_('Reuses')">
-                    <span class="fa fa-retweet fa-fw"></span>
-                    {{ dataset.metrics.reuses || 0 }}
-                </a>
+            <li v-if="dataset.spatial && dataset.spatial.zones && dataset.spatial.zones.length > 0" v-tooltip :title="_('Territorial coverage')">
```

Comment:
no need for `> 0` I think

Reply:
This does not seem to display anything: <img width="861" alt="capture d ecran 2018-02-02 11 54 15" src="https://user-images.githubusercontent.com/119625/35729705-defe3f8c-080f-11e8-81a3-fe8d66f27b35.png">